src/selectors/hosts.ts

```diff
@@ -7,7 +7,21 @@ export const getHosts = (state: RootState): Host[] => state.hosts.hosts;
 export const getHostsLoading = (state: RootState): boolean =>
   state.hosts.loading;
 
+const hostToHostTableRow = (host: Host): string[] => {
+  const spec = host.spec ? host.spec : {};
+  const status = host.status ? host.status : {};
```

Comment:
these 2 lines can be shortened to:   `const { spec = {}, status = {} } = host;`

Reply:
Done